./gradlew liquibaseDiffChangelog throws Errors:   Unknown parameter: 'excludeObjects'

Environment : 
 DB: Postgres 9.3
 Build : Maven
Java Version 8
OS : Mac OSX 10.10
./gradlew liquibaseDiffChangelog command throws 

:compileJava UP-TO-DATE
:nodeSetup SKIPPED
:npmSetup SKIPPED
:npmInstall UP-TO-DATE
:bower
:processResources UP-TO-DATE
:classes UP-TO-DATE
:findMainClass
:liquibaseDiffChangelog
Errors:
  Unknown parameter: 'excludeObjects'
Last few lines of output : 
Unexpected error running Liquibase: Unknown parameter: 'excludeObjects'

[ERROR] liquibase - Unknown parameter: 'excludeObjects'
liquibase.exception.CommandLineParsingException: Unknown parameter: 'excludeObjects'
	at liquibase.integration.commandline.Main.parseOptions(Main.java:702) [liquibase-core-3.3.2.jar:na]
	at liquibase.integration.commandline.Main.run(Main.java:124) [liquibase-core-3.3.2.jar:na]
	at liquibase.integration.commandline.Main.main(Main.java:94) [liquibase-core-3.3.2.jar:na]


For more information, use the --logLevel flag
:liquibaseDiffChangelog FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':liquibaseDiffChangelog'.
> Process 'command '/Library/Java/JavaVirtualMachines/jdk1.8.0_31.jdk/Contents/Home/bin/java'' finished with non-zero exit value 255

Found this issue https://liquibase.jira.com/browse/CORE-875 and found it to be fixed in 3.3.2 .

--- a/app/templates/_liquibase.gradle
+++ b/app/templates/_liquibase.gradle
@@ -19,8 +19,8 @@ task liquibaseDiffChangelog(dependsOn: compileJava, type: JavaExec) {
   args "--password="
   args "--url=<% if (devDatabaseType == 'mysql') { %>jdbc:mysql://localhost:3306/<%= baseName %><% } else if (devDatabaseType == 'postgresql') { %>jdbc:postgresql://localhost/<%= baseName %><% } %>"
   args "--driver=<% if (devDatabaseType == 'mysql') { %>com.mysql.jdbc.Driver<% } else if (devDatabaseType == 'postgresql') { %>org.postgresql.Driver<% } %>"
-  <% if (authenticationType == 'oauth2') { %>args "--excludeObjects=oauth_access_token, oauth_approvals, oauth_client_details, oauth_client_token, oauth_code, oauth_refresh_token"<% } %>
   args "diffChangeLog"
+  <% if (authenticationType == 'oauth2') { %>args "--excludeObjects=oauth_access_token, oauth_approvals, oauth_client_details, oauth_client_token, oauth_code, oauth_refresh_token"<% } %>
 
 }
 

